frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -221,6 +221,11 @@ action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message.HTML=
   </html>
 
 
+action.GitMachete.TraverseFromSelectedAction.text=Git Machete: Traverse from Selected Branch\u2026
+action.GitMachete.TraverseFromSelectedAction.GitMacheteContextMenu.text=T_raverse from Selected Branch\u2026
+action.GitMachete.TraverseFromSelectedAction.description=Traverse from selected branch
```

Comment:
Hmm

<img width="448" alt="image" src="https://user-images.githubusercontent.com/3383210/205462724-1c0c1fe6-cc54-45b9-b17c-39bad1ba6d27.png">

It's now the sole action in context menu with `...from Selected Branch` qualifier (even though all others act on the selected branch in some way &mdash; this "acting on selected branch" is implied).

Maybe hmm... `Traverse from Here...`?

Reply:
Also, small inconsistency: `In` and `Out` after `Slide` are the sole capitalized prepositions... but maybe that's intentional